scripts/superdesk-archive/views/upload.html

```diff
@@ -23,7 +23,7 @@ <h3 class="pull-left">{{ activity.label|translate }}</h3>
         </div>
         <ul class="upload-thumbs">
             <li ng-repeat="item in items">
-                <div class="thumb" sd-image-preview="item.url" data-file="item.file">
+                <div class="thumb" sd-image-preview="item.url" data-file="item.file" data-width="item.width" data-height="item.height">
```

Comment:
why do you need this ?
Can't you directly use the values from the scope since you compute them is the same directive ?